vector/src/main/res/layout/typing_message_layout.xml

```diff
@@ -1,38 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:paddingTop="2dp">
 
+    <!-- This is a LinearLayout which will contain avatars of the typing users -->
```

Comment:
this comment feels like it could be replaced by renaming the id to `typingUsersAvatars`

Reply:
Good point, done

Reply:
(I keep the comment since I added it for clarity when looking at the layout. Since there is no avatar preview)